### PR TITLE
[CHI-194] Add fluid support to button component

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

**Comment by jllr** [comment]
@davidojedacl can you add it in the documentation too?

**Comment by davidojedacl** [comment]
done

File: src/chi/components/buttons/buttons.scss [hunk 1]
```diff
@@ -263,6 +263,10 @@ $sizes: (
         }
       }
 
+      &.-fluid:not(.-float) {
+        width: 100%;
```
Comment (mattnickles) [inline src/chi/components/buttons/buttons.scss]: @davidojedacl Text should be centered by default. `justify-content: center;`

Comment (jllr) [reply src/chi/components/buttons/buttons.scss]: @mattnickles why don't we leave left alignment by default and then using utilities classes we can override the left alignment and center the text. And we document it in the button fluid area?

Comment (mattnickles) [reply src/chi/components/buttons/buttons.scss]: @jllr 👍

**Pull request merged**